### PR TITLE
#143 MWE metric has impediments

### PR DESCRIPTION
**Pull request by uurl**
For #143

**Issue**
MWE metric has impediments. Need to wait for resolving these impediments, add MWE to reports list and cover it by tests.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#240](https://github.com/yegor256/jpeek/issues/240) is now in scope, role is `REV`

**Comment by uurl** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` are banned at [#240](https://github.com/yegor256/jpeek/issues/240); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@uurl are you sure we haven't seen these changes before?

**Comment by uurl** [comment]
@yegor256 yes, these changes are new

File: src/main/java/org/jpeek/App.java [hunk 1]
```diff
@@ -68,12 +68,12 @@
  * @checkstyle MethodLengthCheck (500 lines)
  * @checkstyle JavaNCSSCheck (500 lines)
  *
- * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xml).
+ * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xsl).
```
Comment (yegor256) [inline src/main/java/org/jpeek/App.java]: @uurl I've seen this line in 5 different pull requests. Please, use separate branches.

**Comment by yegor256** [comment]
@uurl please, use separate branches

**Comment by 0crat** [comment]
The job [#240](https://github.com/yegor256/jpeek/issues/240) is now out of scope

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order

**Pull request closed without merging**